### DISCUSSION TITLE
Fensterzustand TRV

### DISCUSSION THREAD

**Discussion by Oderik** [SukramJ/aiohomematic#725] [Q&A] [answered]
Ich wünsche mir Lese- und Schreibzugriff auf den Fenster-Zustand in Thermostaten (ich habe diverse HmIP-eTRV-B).

Die verwandte Diskussion habe ich gesehen: https://github.com/danielperna84/hahomematic/discussions/658 Die Lösung mit dem set_device_value Service habe ich bereits eine Weile im Einsatz. In letzter Zeit habe ich hier allerdings Probleme. Außerdem fehlt hier das Lesen des entsprechenden Zustands.

In der Raspberrymatic UI wird direkt für Thermostate ein Fensterzustand offen / geschlossen angezeigt. Daher nehme ich an, dass man diesen aus Homematic extrahieren kann. Wünschenswert wäre das aus meiner Sicht (Ermittlung des Zustands nach Neustart, eventuell durch Automatik des Thermostats veränderter Zustand, Übertragungsfehler...)

Das Setzen des Zustands über set_device_value ist ein gangbarer aber sehr unschöner Weg.

Ich würde mir für den Fenster-Zustand eines Thermostats entweder eine separate Entität / Gerät wünschen (z.B. ein Schalter) oder eine Bereitstellung über die climate Entität (Attribut / Preset / HVAC Mode?).

PS: Vielen Dank für die Integration bis hierher!

**Comment by SukramJ** [comment, marked as answer]
Lies dir den [Abschnitt zu unignore](https://github.com/danielperna84/custom_homematic#unignore-device-parameters) in der Doku durch.

Der Parameter ist `WINDOW_STATE`.

**Reply by Oderik** [reply]
Wahnsinn, ich bin begeistert! Vielen lieben Dank!!

SOLVED
